kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorTest.java

```diff
@@ -347,52 +347,55 @@ public void testITEMCOMP_DUPLICATED() {
     @Test
     public void testITEMDEF_NOT_UNIQUE() {
         List<DMNMessage> validate = validator.validate( getReader( "ITEMDEF_NOT_UNIQUE.dmn" ), VALIDATE_SCHEMA, VALIDATE_MODEL, VALIDATE_COMPILATION);
-        System.out.println( formatMessages( validate ) );
         assertThat( formatMessages( validate ), validate.size(), is( 2 ) );
         assertTrue( validate.stream().anyMatch( p -> p.getMessageType().equals( DMNMessageType.DUPLICATED_ITEM_DEF ) ) );
     }
     
     @Test
     public void testITEMDEF_NOT_UNIQUE_DROOLS_1450() {
         // DROOLS-1450
-        Definitions definitions = utilDefinitions( "ITEMDEF_NOT_UNIQUE_DROOLS-1450.dmn", "ITEMDEF_NOT_UNIQUE" );
-        List<DMNMessage> validate = validator.validate(definitions);
-        
-        assertFalse( validate.stream().anyMatch( p -> p.getMessageType().equals( DMNMessageType.DUPLICATED_ITEM_DEF ) ) );
+        List<DMNMessage> validate = validator.validate( getReader( "ITEMDEF_NOT_UNIQUE_DROOLS-1450.dmn" ), VALIDATE_SCHEMA, VALIDATE_MODEL, VALIDATE_COMPILATION);
+        assertThat( formatMessages( validate ), validate.size(), is( 0 ) );
```

Comment:
I think this is "dangerous", the DMN file is constructed to demonstrate 1 specific issue, but we shouldn't ensure the DMN file per-se is valid 100% with respect to other potential problems?
So the original check was to demonstrate it does NOT pose any `DMNMessageType.DUPLICATED_ITEM_DEF` problem